dhis-2/dhis-api/src/main/java/org/hisp/dhis/program/ProgramIndicatorService.java

```diff
@@ -190,6 +190,7 @@
      * @param programIndicator the program indicator to evaluate.
      * @param startDate the start date.
      * @param endDate the end date.
+     * @param ignoreMissingValues replace nulls with 0 or ''.

```

Comment:
I wonder if anyone still calls with ignoreMissingValues = false, or if we can remove this?